webapp/tests/forms/steps/test_eligibility_steps.py

```diff
@@ -1848,39 +1589,6 @@ def test_if_joint_taxes_user_b_no_elster_account_session_data_correct_then_set_p
             step.handle()
         self.assertEqual(expected_url, step.render_info.prev_url)
 
-    def test_if_elster_registration_method_unknown_session_data_correct_then_set_prev_input_step_correctly(self):
-        alternative_data = {'marital_status_eligibility': 'married',
-                            'separated_since_last_year_eligibility': 'no',
-                            'joint_taxes_eligibility': 'yes',
```

Comment:
Could you instead add tests for the newly added possible previous steps (user_a_has_elster_account and user_b_has_elster_account)?